modules/gis/src/lib/reproject.js

```diff
@@ -0,0 +1,36 @@
+export function mapBinaryCoords(binaryFeatures, fn) {
+  // Expect binaryFeatures to have points, lines, and polygons keys
+  for (const binaryFeature of Object.values(binaryFeatures)) {
+    const {positions} = binaryFeature;
+    for (let i = 0; i < positions.value.length; i += positions.size) {
+      positions.value.set(fn(Array.from(positions.value.subarray(i, i + positions.size))), i);
+    }
+  }
+  return binaryFeatures;
+}
+
+export function mapGeoJsonCoords(features, fn) {
+  for (const feature of features) {
```

Comment:
Total nit: This might actually read a little easier if it wasn't split into so many functions. And that comes from someone who often asks for smaller functions...